region/crateria/central.json

```diff
@@ -1122,13 +1196,23 @@
           "name": "Left Door",
           "nodeType": "door",
           "nodeSubType": "grey",
-          "unlock": [
-            {"or": [
-              {"not": [
+          "locks": [
+            {
+              "lock": [
                 "Bombs"
-              ]},
-              "DefeatBombTorizo"
-            ]}
+              ],
+              "unlock": [
+                "DefeatBombTorizo"
+              ]
+            },
+            {
+              "lock": [
```

Comment:
Since the door's second lock is here, did you intend to also add an animals node also locked by `ZebesSetAblaze` that yields `AnimalsSaved`?